homeassistant/components/tradfri/base_class.py

```diff
@@ -49,25 +52,6 @@ def _async_start_observe(self, exc=None):
         """Start thread when added to hass."""
         self._async_start_observe()
 
-    @property
-    def available(self):
```

Comment:
Why aren't we implementing this property for the base class? Don't groups support that?

Reply:
For some reason groups doesn't have that today. Let's move it to the base class and see if we can implement it.

Reply:
(I didn't make the GroupLight class so I'm not quite sure how it works yet.)